Automattic/jetpack
# CLI: Refactor unit tests

Existing unit tests are only unit tests, but wanting to keep separation from integration tests with the yargs lib that provides the UI for the CLI.

#### Changes proposed in this Pull Request:

* Adds initial integration tests.

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* `cd tools/cli && yarn install` (since the CLI is a separate project, need to install the devdeps)
* `yarn test` (should run all tests)
* `yarn test --type all` (same as above)
* `yarn test --type unit` (only runs the unit tests)
* `yarn test --type integration` (only runs the integration tests)

#### Proposed changelog entry for your changes:


* n/a
